File: nightgrass/src/main/java/com/goodfeel/nightgrass/data/ProductPhoto.java
package com.goodfeel.nightgrass.data;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("e_mall_product_photo")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ProductPhoto {
    @Id
    private Long id;
    private Long productId;
    private String photoUrl;
}

File: nightgrass/src/main/kotlin/com/goodfeel/nightgrass/data/ProductPhoto.kt
package com.goodfeel.nightgrass.data

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("e_mall_product_photo")
data class ProductPhoto(
    @Id
    val id: Long? = null,
    val productId: Long,
    val photoUrl: String
)
